grakn-graph/src/test/java/ai/grakn/graph/internal/GraknGraphPropertyIT.java

```diff
@@ -617,33 +625,10 @@ public void whenCallingClear_TheGraphCloses(@Open GraknGraph graph) {
         assertTrue(graph.isClosed());
     }
 
-    @Ignore // TODO: Re-enable this when test below is fixed
-    @Property
-    public void whenCallingClear_OnlyMetaConceptsArePresent(@Open GraknGraph graph) {
-        graph.clear();
-        graph.open();
-
-        List<Concept> concepts = allConceptsFrom(graph);
-        concepts.forEach(concept -> {
-            assertTrue(concept.isType());
-            assertTrue(isMetaName(concept.asType().getName()));
-        });
-    }
-
-    @Ignore // TODO: Fix this
-    @Property
-    public void whenCallingClear_AllMetaConceptsArePresent(
-            @Open GraknGraph graph, @From(MetaTypeNames.class) TypeName typeName) {
-        graph.clear();
-        graph.open();
-        assertNotNull(graph.getType(typeName));
-    }
```

Comment:
Instead of deleting these, can we leave them as ignored?

Although they will have to change with the new transaction behaviour, the behaviour they test for is still expected.